lib/tests/modules/graph/test.nix

```diff
@@ -0,0 +1,50 @@
+let
+  lib = import ../../..;
+
+  evaluation = lib.evalModules {
+    modules = [
```

Comment:
maybe use some modules with explizit `key` attribute, then you can check where and if they appear in the output ?

Reply:
Okay, done.